SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialReceiver.h

```diff
@@ -37,13 +37,6 @@ struct PendingAddComponentWrapper
 	PendingAddComponentWrapper(Worker_EntityId InEntityId, Worker_ComponentId InComponentId, TUniquePtr<SpatialGDK::DynamicComponent>&& InData)
 		: EntityId(InEntityId), ComponentId(InComponentId), Data(MoveTemp(InData)) {}
 
-	// We define equality to cover just entity and component IDs since duplicated AddComponent ops
-	// will be moved into unique pointers and we cannot equate the underlying Worker_ComponentData.
-	bool operator==(const PendingAddComponentWrapper& Other) const
-	{
-		return EntityId == Other.EntityId && ComponentId && Other.ComponentId;
```

Comment:
Wait - this is the bad code
`return EntityId == Other.EntityId && ComponentId && Other.ComponentId;`
should be
`return EntityId == Other.EntityId && ComponentId == Other.ComponentId;`

Reply:
rofl, what a moron